File: src/test/scala/cern/colt/matrix/tdouble/impl/SparseRCMDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.SparseRCMDoubleMatrix2D

class SparseRCMDoubleMatrix2DViewTest(arg0: String) extends SparseRCMDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseRCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseRCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseRCMDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tdouble/impl/SparseRCMDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class SparseRCMDoubleMatrix2DViewTest extends SparseRCMDoubleMatrix2DTest {

    public SparseRCMDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCMDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCMDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
